### DISCUSSION TITLE
Export query to insert sql statement with te alias of columns

### DISCUSSION THREAD

**Discussion by franbar95** [dbeaver/dbeaver#19762] [Ideas]
Dear dbeaber team,

I would like to request a new feature for the dbeaber tool. Specifically, I would like the ability to export SQL insert statements with the SQL alias name of columns.

Currently, when exporting data to SQL insert statements, the tool uses the original column names as they appear in the table. However, in some cases, the original column names may not be very descriptive or easy to understand. Using SQL alias names in the insert statements would help make the code more readable and easier to understand for other developers.

In example when export the query `SELECT id, name AS product, price FROM products` as insert SQL statement results in `INSERT INTO table_name (id, name, price)` instead of `INSERT INTO table_name (id, product, price)`

Therefore, it would be very helpful if dbeaber could include an option to export SQL insert statements with SQL alias names. This could be achieved by adding a checkbox in the export dialog box that allows the user to choose whether to use the alias names or the original column names.

Thank you for considering my request. I believe this feature would greatly improve the user experience for developers who use dbeaber.

Sincerely,
Franbar

**Comment by ShadelessFox** [comment]
Thanks for the feature request.

**Comment by jalesc** [comment]
10/04/2024 and nothing... exporting with jSON that work.
